src/main/java/com/worksap/nlp/sudachi/dictionary/WordIdTable.java

```diff
@@ -43,4 +45,23 @@ int storageSize() {
         }
         return result;
     }
+
+    int fillBuffer(int index, WordLookup lookup) {
```

Comment:
The name of this method is confusing, so please change it so that we can understand what it is intended to do.

Reply:
fixed